docs/data/date-pickers/custom-views/MonthButtonComponentProps.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { DemoContainer } from '@mui/x-date-pickers/internals/demo';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import { DateCalendar } from '@mui/x-date-pickers/DateCalendar';
+
+export default function MonthButtonComponentProps() {
+  return (
+    <LocalizationProvider dateAdapter={AdapterDayjs}>
+      <DemoContainer components={['DateCalendar']}>
+        <DateCalendar
+          slotProps={{
+            monthButton: {
+              sx: { border: '1px red solid' },
```

Comment:
I started with a demo as light as possible so that the preview works.
But yours is of course more good-looking.

If you think the preview is not important here then I can definitely take the style from your screenshot.